karma_chameleon/bot.py

```diff
@@ -156,6 +175,9 @@ def decrement_karma(self, msg: dict) -> str:
             self.logger.debug("Skipping self-decrement")
             return "Now, now.  Don't be so hard on yourself!"
 
+        if self._check_for_url(msg):
+            return ""  # Fail silently... no need to respond to the user.
```

Comment:
It's not _technically_ a failure state here. We should probably just return `None` here instead of an empty string as well.